scipy/stats/_qmc.py

```diff
@@ -336,10 +336,57 @@ def discrepancy(
                          f" {set(methods)!r}")
 
 
+def pmindist(
+        sample: "npt.ArrayLike",
+        metric: str = "euclidean",
+        workers: IntNumber = 1) -> float:
+    """Mininum distance between points in the given sample.
+
+    Parameters
+    ----------
+    sample : array_like (n, d)
+        The sample to compute the minimum distance from.
+    metric : str or callable, optional
+        The distance metric to use. See the documentation
+        for `scipy.spatial.distance.pdist` for the available metrics and
+        the default.
+    workers : int, optional
+        Number of workers to use for parallel processing. If -1 is given all
+        CPU threads are used. Default is 1.
+
+    Returns
+    -------
+    distance : float
+        Minimum distance.
+        
+    """
+    if metric == 'euclidean':
+        p = 2
+        distance_fun = distance.euclidean
+    elif metric == 'cityblock':
+        p = 1
+        distance_fun = distance.cityblock
+    elif metric == 'chebyshev':
+        p = np.inf
+        distance_fun = distance.chebyshev
+    else:
+        # Slow path for metrics unsupported by KDTree.
+        distances = distance.pdist(sample, metric=metric)  # type: ignore[call-overload]
+        return distances.min()
+
+    distance_upper_bound = distance_fun(sample[0,...], sample[1,...])
+    tree = KDTree(sample)
+    d, _ = tree.query(sample,
+                      k=[2], p = p,
```

Comment:
minor/nit: remove spacing between `p = p`, which anyway would be consistent with the rest of your formatting and what we usually do

Reply:
OK. I was hoping that the automatic pre-commit linter (like ruff) would fix those automatically.